packages/docs/src/routes/qwikcity/api/index.mdx

```diff
@@ -74,12 +74,9 @@ Use the [`useLocation()`](/qwikcity/api/index.mdx#uselocation) function to retri
 
 ```tsx
 export interface RouteLocation {
-  readonly params: RouteParams; // Record<string,string>
-  readonly href: string;
-  readonly pathname: string;
-  readonly query: Record<string, string>;
-  readonly isNavigating: boolean;
+  readonly params: RouteParams; // Readonly<Record<string, string>>
   readonly url: URL;
+  readonly isNavigating: boolean;
```

Comment:
Switched positions of `url` and `isNavigating` since `url` is gonna be much more used property, plus [it](https://github.com/BuilderIO/qwik/blob/9cbb12d7915865ed1b3dfb12bf74f45cd56c9e84/packages/qwik-city/runtime/src/types.ts#L60) is positioned before [`isNavigating`](https://github.com/BuilderIO/qwik/blob/9cbb12d7915865ed1b3dfb12bf74f45cd56c9e84/packages/qwik-city/runtime/src/types.ts#L73) in the interface declaration.